### ISSUE TITLE
Transifex updates should be on translated or reviewed

### ISSUE THREAD

**Issue by ylavoie** [lumoslabs/txgh#40]
Having to wait on a fully translated for integration with github is counter-productive. There should be a way to have Transifex push strings as they get translated or reviewed.

**Comment by camertron** [comment]
Agreed. In my conversations with Transifex they were hesitant to enable such behavior since the webhook could potentially fire very often (and overwhelm any receiver). As far as I know there's still no option to receive more frequent webhooks. At Lumos we solved this by creating a cron job that pulls every hour or so.

**Comment by ylavoie** [comment]
From what I understand of my recent discussions with them, they are working with Transifex to have such hook implemented soon.

**Comment by camertron** [comment]
That's good news :)

**Comment by ylavoie** [comment]
TXGH is set, they waiting for Transifex to complete coding on their side.
